src/components/BreedPhotos/BreedPhotos.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import ReactPaginate from 'react-paginate';
+import ImageCard from '../ImageCard/ImageCard';
+import formatBreedName from '../../helpers/formatName';
+import { BreedPhotosGrid, StyledMain, StyledPaginateContainer } from './styles';
+
+type BreedPhotosProps = {
```

Comment:
Why types instead of interfaces?

Reply:
I don't have extensive experience with TS, but when I was building my implementation for dogLover, I googled about types vs interfaces and read a couple of articles describing that both were quite similar and that using one or another depends on if you wish to use the declaration merging feature of interfaces. I ended up using types thinking it does not matter for my use case.

Links here:
[1](https://blog.logrocket.com/types-vs-interfaces-in-typescript/)
[2](https://stackoverflow.com/questions/37233735/interfaces-vs-types-in-typescript)